packages/tools/src/tools/annotation/ProbeTool.ts

```diff
@@ -92,7 +92,7 @@ const { transformWorldToIndex } = csUtils;
  */
 
 class ProbeTool extends AnnotationTool {
-  static toolName;
+  static toolName = 'Probe';
```

Comment:
this will cause some issues in some evns i remember, can we keep it outside like before?

Reply:
It used to cause issues because of the build, but in 2.x it is fine.

Reply:
I mean either change all or none, i rather have one PR to change all, than spread them into separate PRs

Reply:
Changed all.